libr/bin/p/bin_mz.c

```diff
@@ -58,22 +58,20 @@ static bool check_bytes(const ut8 *buf, ut64 length) {
 		}
 
 		// Check for New Executable, LE/LX or Phar Lap executable
-		if (!memcmp (buf + new_exe_header_offset, "NE", 2) ||
-			!memcmp (buf + new_exe_header_offset, "LE", 2) ||
+		if (!memcmp(buf, "MZ", 2) && 
+			!memcmp (buf + new_exe_header_offset, "NE", 2) ||
 			!memcmp (buf + new_exe_header_offset, "LX", 2) ||
 			!memcmp (buf + new_exe_header_offset, "PL", 2)) {
-			if (!checkEntrypoint (buf, length)) {
-				return false;
-			}
+            	return false;
 		}
 	}
-
-	// Raw plain MZ executable (watcom)
+	
 	if (!checkEntrypoint (buf, length)) {
 		return false;
 	}
-
+	
```

Comment:
You added unneeded tab here - please check the code for right indentation and trailing whitespaces before committing.

Reply:
sorry, my bad. I will make sure to check the code for tabs and unnecessary spaces before committing next time.